cachelib/cachebench/cache/Cache.h

```diff
@@ -227,9 +230,15 @@ class Cache {
   // @param keys  list of keys that the stressor uses for the workload.
   void enableConsistencyCheck(const std::vector<std::string>& keys);
 
+  // enables validating all values on find. Each value is compared to
+  // expected Value.
+  void enableValueValidating(const std::string& expectedValue);
```

Comment:
In case of consistency checking, the enable happens later since the keys are generated by inserting into the cache and then we enable the consistency checking for the keys added to the cache. It is a bit convoluted, but we don't need to do the same.

Reply:
Inside Cache ctor we do not know what is the `expectedValue` yet. This is why I call `enableValueValidating` from CacheStressor which is responsible for generating the values (`hardcodedString_` for now).

Reply:
I removed this function and added bool param to the Cache ctor - is it OK?